Recurly/Resources/AddOn.cs

```diff
@@ -55,9 +55,10 @@ public class AddOn : Resource
         [JsonProperty("plan_id")]
         public string PlanId { get; set; }
 
-        /// <value>Add-ons can be either active or inactive.</value>
+        /// <value>Active state can be either active or inactive.</value>
```

Comment:
Changes like this happened because the `ActiveState` enum is used in multiple places so I adjusted the description to be more generic because it's defined on the enum schema.